eng/nuget/Microsoft.NET.Workload.Emscripten.net7.Manifest/WorkloadManifest.targets

```diff
@@ -1,28 +1,39 @@
 <Project>
-  <PropertyGroup Condition="'$(RuntimeIdentifier)' == 'browser-wasm'">
-    <BrowserWorkloadDisabled Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and !$([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)', '6.0'))">true</BrowserWorkloadDisabled>
+  <PropertyGroup>
+    <_EmscriptenWorkloadDisabled7>$(BrowserWorkloadDisabled)</_EmscriptenWorkloadDisabled7>
+    <_EmscriptenWorkloadDisabled7 Condition="'$(_EmscriptenWorkloadDisabled7)' == '' and
+                                              '$(RuntimeIdentifier)' == 'browser-wasm' and
+                                              '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and
+                                              !$([MSBuild]::VersionEquals('$(TargetFrameworkVersion)', '7.0'))">true</_EmscriptenWorkloadDisabled7>
+    <TargetsNet7 Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and $([MSBuild]::VersionEquals('$(TargetFrameworkVersion)', '7.0'))">true</TargetsNet7>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(RuntimeIdentifier)' == 'browser-wasm' and '$(BrowserWorkloadDisabled)' == 'true'">
-    <_NativeBuildNeeded Condition="'$(RunAOTCompilation)' == 'true'">true</_NativeBuildNeeded>
-    <WorkloadDisabledWithReason Condition="'$(_NativeBuildNeeded)' == 'true'">WebAssembly workloads (required for AOT) are only supported for projects targeting net6.0+</WorkloadDisabledWithReason>
+  <PropertyGroup Condition="'$(TargetsNet7)' == 'true'">
+    <!-- if UsingBrowserRuntimeWorkload has already been set, for example, maybe the wasm manifest
+         decided that we need it. -->
+    <_UsingEmscriptenWorkload7>$(UsingBrowserRuntimeWorkload)</_UsingEmscriptenWorkload7>
+    <WasmWorkloadNeeded Condition="'$(RunAOTCompilation)' == 'true' or '$(WasmEnableSIMD)' == 'true' or
```

Comment:
This condition looks like it when not be automatically true when the workload is installed

Reply:
you probably want $(WasmNativeWorkload) == 'true' for 6, and or ($WasmNativeWorkload) != 'false' and $(WasmNativeWorkload7) == 'true' in 7